electrum/gui/qt/transaction_dialog.py

```diff
@@ -947,228 +919,3 @@ def __init__(self, tx: Transaction, *, parent: 'ElectrumWindow', desc, prompt_if
-        #
-        self.needs_update = True
-
-    def update_fee_fields(self):
-        freeze_fee = self.is_send_fee_frozen()
-        freeze_feerate = self.is_send_feerate_frozen()
-        tx = self.tx
-        if self.no_dynfee_estimates and tx:
-            size = tx.estimated_size()
-            self.size_e.setAmount(size)
-        if self.not_enough_funds or self.no_dynfee_estimates:
-            if not freeze_fee:
-                self.fee_e.setAmount(None)
-            if not freeze_feerate:
-                self.feerate_e.setAmount(None)
-            self.feerounding_icon.setVisible(False)
-            return
-
-        assert tx is not None
-        size = tx.estimated_size()
-        fee = tx.get_fee()
-
-        self.size_e.setAmount(size)
-        fiat_fee = self.main_window.format_fiat_and_units(fee)
-        self.fiat_fee_label.setAmount(fiat_fee)
-
-        # Displayed fee/fee_rate values are set according to user input.
-        # Due to rounding or dropping dust in CoinChooser,
-        # actual fees often differ somewhat.
-        if freeze_feerate or self.fee_slider.is_active():
-            displayed_feerate = self.feerate_e.get_amount()
-            if displayed_feerate is not None:
-                displayed_feerate = quantize_feerate(displayed_feerate)
-            elif self.fee_slider.is_active():
-                # fallback to actual fee
-                displayed_feerate = quantize_feerate(fee / size) if fee is not None else None
-                self.feerate_e.setAmount(displayed_feerate)
-            displayed_fee = round(displayed_feerate * size) if displayed_feerate is not None else None
-            self.fee_e.setAmount(displayed_fee)
-        else:
-            if freeze_fee:
-                displayed_fee = self.fee_e.get_amount()
-            else:
-                # fallback to actual fee if nothing is frozen
-                displayed_fee = fee
-                self.fee_e.setAmount(displayed_fee)
-            displayed_fee = displayed_fee if displayed_fee else 0
-            displayed_feerate = quantize_feerate(displayed_fee / size) if displayed_fee is not None else None
-            self.feerate_e.setAmount(displayed_feerate)
-
-        # show/hide fee rounding icon
-        feerounding = (fee - displayed_fee) if (fee and displayed_fee is not None) else 0
-        self.set_feerounding_text(int(feerounding))
-        self.feerounding_icon.setToolTip(self.feerounding_text)
-        self.feerounding_icon.setVisible(abs(feerounding) >= 1)
-
-    def can_finalize(self):
-        return (self.tx is not None
-                and not self.not_enough_funds)
-
-    def on_finalize(self):
-        if not self.can_finalize():
-            return
-        assert self.tx
-        self.finalized = True
-        self.stop_editor_updates()
-        self.tx.set_rbf(True)
-        locktime = self.locktime_e.get_locktime()
-        if locktime is not None:
-            self.tx.locktime = locktime
```

Comment:
This code needs to be replicated somewhere. `locktime_e` is being ignored without it.
(should work in all subclasses of TxEditor)

Reply:
fixed in 8c001883485f035c07e82e3e330078aecc855d48